File: HutHelper/Views/Main/PointView.m
//
//  PointView.m
//  HutHelper
//
//  Created by Nine on 2017/7/16.
//  Copyright © 2017年 nine. All rights reserved.
//

#import "PointView.h"

@implementation PointView
- (id)initWithFrame:(CGRect)frame {
    
    self = [super initWithFrame:frame];
    if (self) {
        // Initialization code.
    }
    self.backgroundColor=[UIColor clearColor];
    return self;
}

- (void)drawRect:(CGRect)rect {

    CGContextRef context =UIGraphicsGetCurrentContext();
    
    CGContextAddArc(context, SYReal(10),SYReal(10), SYReal(3), 0,SYReal(3)*3.14, 0); //添加一个圆
    
    CGContextSetFillColorWithColor(context,  [UIColor whiteColor].CGColor);//填充颜色
    CGContextDrawPath(context, kCGPathFill);//绘制填充

}


@end

File: HutHelper/SwiftApp/Main/LeftView/PointView.swift
//
//  PointView.swift
//  HutHelperSwift
//
//  Created by 张驰 on 2020/1/14.
//  Copyright © 2020 张驰. All rights reserved.
//

import UIKit

class PointView: UIView {

    override init(frame: CGRect) {
        super.init(frame: frame)
        self.backgroundColor = .clear
    }
    
    required init?(coder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
    
    // Only override draw() if you perform custom drawing.
    // An empty implementation adversely affects performance during animation.
    override func draw(_ rect: CGRect) {
        // Drawing code
        let context = UIGraphicsGetCurrentContext()
        context?.addArc(center: CGPoint(x: 10.fit, y: 10.fit), radius: 3.fit, startAngle: 0, endAngle: 3.fit * 3.14, clockwise: false) //添加一个圆
        context?.setFillColor(UIColor.white.cgColor) //填充颜色
        context?.drawPath(using: .fill) //绘制填充
    }
    

}
